modules/microshift-creating-backups.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-creating-backups_{context}"]
+= Creating backups
+
+Use the following procedure to create backups compatible with the auto-recovery feature.
+
+.Prerequisites
+
+. {microshift-short} must be stopped.
+
+.Procedure
+
+. Create backups in the parent directory by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift backup --auto-recovery _<path_of_parent_directory>_ <1>
+----
+<1> Replace `_<path_of_parent_directory>_` with the path of the parent directory which contains the auto-recovery file. For example, `/var/lib/microshift-auto-recovery`.
```

Comment:
I would present it as "directory path for storing backups" or similar.
Also, there's no "auto-recovery file"

Reply:
done